src/system/post-office.ts

```diff
@@ -369,7 +369,7 @@ class PostOffice {
      */
     request(event: EventEnvelope, timeout = 60000): Promise<EventEnvelope> {
         return new Promise((resolve, reject) => {
-            const start = performance.timeOrigin + performance.now();
```

Comment:
the performance.timeOrigin + performance.now() returns the current timestamp.

for performance metrics, we are not interested in current time. We only need the diff between 2 points (end - start). Therefore we remove the performance.timeOrigin value.